products/datagrid/get-started.adoc

```diff
@@ -5,77 +5,75 @@
 
 === Install Java
 
-1. Install the Java SE Development Kit (JDK), we recommend OpenJDK or Oracle JDK.
-a. For windows and OS X: Red Hat Developers typically downloads from Oracle’s link:http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html[site].  (Make sure you get the Java *Development* Kit and not just the Java Runtime Environment).
-b. For Red Hat Enterprise Linux:  Install OpenJDK by executing the following command:
+1. Install the Java SE Development Kit (JDK), we recommend OpenJDK 1.8 or Oracle JDK 1.8.
+a. For Windows and Mac: Red Hat Developers typically downloads from Oracle’s link: http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html[site].  (Make sure you get the Java *Development* Kit and not just the Java Runtime Environment).
+b. For Red Hat Enterprise Linux: Install OpenJDK by executing the following command:
 +
-`yum install java-1.7.0-openjdk-devel`
+`yum install java-1.8.0-openjdk-devel`
 +
-TIP: You can link:https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6/html/Installation_Guide/Install_OpenJDK_on_Red_Hat_Enterprise_Linux.html[learn more] about installing OpenJDK on RHEL at the customer portal.
 
-=== Install and Configure JBoss Data Grid
+=== Install and Configure JBoss Data Grid 7
```

Comment:
See: https://www.dropbox.com/s/noco0q6ijp4os56/Screenshot%202016-07-26%2011.39.29.png?dl=0

It looks like the heading is not being set correctly. I've looked at the code on this line and it looks right to me. @LightGuard any ideas?